core/trino-main/src/main/java/io/trino/testing/LocalQueryRunner.java

```diff
@@ -892,6 +893,9 @@ private MaterializedResultWithPlan executeInternal(Session session, @Language("S
             }
 
             verify(builder.get() != null, "Output operator was not created");
+            if (plan.getRoot() instanceof OutputNode outputNode) {
```

Comment:
can this condition be false?

Reply:
All test cases are passing, So I assume check is not required.